src/commands/bundle.ts

```diff
@@ -0,0 +1,76 @@
+import { Flags } from '@oclif/core';
+import { Example } from '@oclif/core/lib/interfaces';
+import Command from '../base';
+import bundle from '@asyncapi/bundler';
+import { promises } from 'fs';
+import path from 'path';
+import { load } from '../models/SpecificationFile';
+
+const { writeFile } = promises;
+
+export default class Bundle extends Command {
+  static description = 'Bundle multiple asyncapi files together.';
+  static strict = false;
+
+  static examples: Example[] = [
+    'asyncapi bundle ./asyncapi.yaml > final-asyncapi.yaml',
+    'asyncapi bundle ./asyncapi.yaml --output final-asyncapi.yaml',
+    'asyncapi bundle ./asyncapi.yaml ./features.yaml --reference-into-components',
+    'asyncapi bundle ./asyncapi.yaml ./features.yaml --base ./asyncapi.yaml --reference-into-components'
+  ];
+
+  static flags = {
+    help: Flags.help({ char: 'h' }),
+    output: Flags.string({ char: 'o', description: 'The output file name. Omitting this flag will create a main.yaml.' }),
+    'reference-into-components': Flags.boolean({ char: 'r', description: 'Bundle the message $refs into components object.' }),
+    base: Flags.string({ char: 'b', description: 'Path to the file which will act as a base. This is required when some properties are to needed to be overwritten.' }),
```

Comment:
```suggestion
    base: Flags.string({ char: 'b', description: 'Path to the file which will act as a base. This is required when some properties need to be overwritten.' }),
```

What `This is required when some properties need to be overwritten` means? I tried to figure from bundle project history why it was added, also looked into test case, but in test cases we do not evaluate bundle against some real result, so I could not figure quickly what is the difference between bundling with and without `base`

Reply:
maybe this could help https://github.com/asyncapi/bundler/issues/101 , this is something most people wouldn't be interested in, but it is like nice to have feature. You can use `base` flag to overwrite or retain info object when you are merging different asyncapi files.

Reply:
when I read explanation, I understand that `base` is basically something that holds basic information that should stay in asyncapi.yml and all the other files are added to it, cannot override what is in base already

but then I see the linked issue and the expected behavior section. Looks like what is there in base is lost and expected result is `This service is in charge of processing user logouts` and not what is in base `This service is in charge of processing user signups`

sorry for all these questions :(